docs/aks-acr-across-tenants.md

```diff
@@ -0,0 +1,59 @@
+# AKS and ACR in different tenants
+
+## Introduction
+
+There are several ways to set up the auth credential in Kubernetes to pull image from ACR. For example, you can use [admin user or repository scoped access token](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal) to configure pod [imagePullSecrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). 
+
+While `imagePullSecrets` is commonly used, it brings the challenge and overhead to manage the corresponding secret. On Azure, you can set up [AKS cluster with a service principal credential](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal) which allows you securely pull the image from ACR without additional `imagePullSecrets` setting on each pod.
+
+Sometimes, you may have your AKS and ACR in different Azure Active Directories (Tenants). This document will walk your through the steps to enable cross tenant authentication using service principal credential.
+
+## Instruction
+
+In this example, the AKS cluster is in `Tenant A` and the ACR is in `Tenant B`. 
+
```

Comment:
What permissions does customer need in Tenant A and Tenant B?

Reply:
updated